### PR TITLE
feat: inject toolbox in navbar

### PR DESCRIPTION
**Pull request by zhouhanseng**
close https://github.com/DimensionDev/Maskbook/issues/2613

### CODE REVIEW

File: packages/maskbook/src/social-network/ui.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export async function activateSocialNetworkUI(): Promise<void> {
     ui.collecting.profilesCollector?.(signal)
     ui.injection.pageInspector?.(signal)
     if (Flags.toolbar_enabled) ui.injection.toolbar?.(signal)
+    ui.injection.toolBoxInNavBar?.(signal)
```
Comment (Jack-Works) [inline packages/maskbook/src/social-network/ui.ts:74]: Should we remove `ui.injection.toolbar`?

File: packages/maskbook/src/social-network/types.ts [hunk 2]
```diff
@@ -103,6 +103,8 @@ export namespace SocialNetworkUI {
             postInspector?(signal: AbortSignal, current: PostInfo): void
             /** Inject a toolbar that displayed on the top of the page */
             toolbar?(signal: AbortSignal): void
+            /** Inject a tool box that displayed in the twitter navigation bar */
```
Comment (Jack-Works) [inline packages/maskbook/src/social-network/types.ts:106]:
```suggestion
            /** Inject a tool box that displayed in the navigation bar of the SNS */
```

This is not only for Twitter.

File: packages/maskbook/src/social-network-adaptor/twitter.com/injection/Toolbox.tsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// import { injectToolboxDialogAtTwitter } from './ToolboxDialog'
+import { injectToolboxHintAtTwitter } from './ToolboxHint'
+
+export function injectToolBoxComposed(signal: AbortSignal) {
```
Comment (Jack-Works) [inline packages/maskbook/src/social-network-adaptor/twitter.com/injection/Toolbox.tsx:4]:
```suggestion
export function injectToolBox(signal: AbortSignal) {
```

File: packages/maskbook/src/social-network/ui.ts (hunk 1 above)
Comment (zhouhanseng) [reply packages/maskbook/src/social-network/ui.ts:74]: Temporarily not.

Comment (Jack-Works) [reply packages/maskbook/src/social-network/ui.ts:74]: should it in a flag?

**Review by Jack-Works — APPROVED** [review]

File: packages/maskbook/src/social-network/ui.ts (hunk 1 above)
Comment (zhouhanseng) [reply packages/maskbook/src/social-network/ui.ts:74]: oh, it requires flags rather than here but for some of its entries,  updated.

Comment (zhouhanseng) [reply packages/maskbook/src/social-network/ui.ts:74]: update

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**